### PR TITLE
Various website fixes (banner, icons, css, articles)

### PR DESCRIPTION
**Pull request by PhilipWitte**
- Fixed website banner HTML & Javascript (starts/rolls correctly & JS only included on the index page)
- Various website CSS improvements (reformatted original and combined with dom96's additions better)
- Added docs & learn icons (more visual candy)
- Added learn articles (took the most popular articles from /r/nim + changed the bootstrapping link)

### CODE REVIEW

**Comment by PhilipWitte** [comment]
Forgot to add: Also made 'community' and 'news' not show up as header items when you're in those pages. I meant to add "Latests News" as a header to the news page (so users would know where they where without the header item) but I forgot to add that to this PR.

**Comment by Araq** [comment]
It's called RST ;-) and styling standout-->ul->li-a href should be easy enough with CSS.

File: web/assets/style.css [hunk 1]
```diff
@@ -551,48 +546,13 @@ pre .end { background:url("images/tabEnd.png") no-repeat left bottom; }
...
-    z-index:2;
-		position:absolute;
-		top:-340px;
-		right:25px;
-		width:202px;
-		height:319px;
-		background:url("images/mascot.png") no-repeat; }
-
-
-#body pre {
```
Comment (dom96) [inline web/assets/style.css]: Why was this removed?
